lambda_proxy/proxy.py

```diff
@@ -235,10 +236,10 @@ def __call__(self, event, context):
 
         view_function = route_entry.view_function
         function_args = self._get_matching_args(route_entry.uri_pattern, resource_path)
-        self.current_request = Request(event['queryStringParameters'], event['path'], event['httpMethod'])
+        self.current_request = Request(event)
 
         try:
-            response = view_function(*function_args)
+            response = view_function(event, context, *function_args)
```

Comment:
@MattReimer I'm not sure about adding `event` and `context` to the function's inputs. The whole point of lambda-proxy was to not have to care about the event nor context object in the handler function.

Reply:
I think we could hack a bit and do something like: 

```
function_kwargs = {}

if event["httpMethod"] == 'POST':
    function_kwargs.update(dict(body=event.get("body", None))) 

response = route_entry.view_function(*function_args, **function_kwargs)
```

and then your handler should looks like: 
```
@APP.route("/test", methods=["POST"], cors=True)
def search(body=None):
```